src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -43,6 +43,28 @@ function getState(wrapper): IEditorPageState {
     return wrapper.find(EditorPage).childAt(0).state() as IEditorPageState;
 }
 
+function getMockAssetMetadata(testAssets, assetIndex= 0): IAssetMetadata {
+    const mockRegion = MockFactory.createMockRegion();
+    return {
+        asset: {
+            ...testAssets[assetIndex],
+            state: AssetState.Visited,
+        },
+        regions: [
+            {
+                ...mockRegion,
+                tags: [
+                    {
+                        ...mockRegion.tags[0],
+                        color: expect.stringMatching(/^#[0-9a-f]{3,6}$/i),
```

Comment:
how does expect.stringMatching works here?

Reply:
Rather than having a specific string it's expecting, I can give it `expect.stringMatching` with a regex and then use `toMatchObject` to see if the property matches the condition (the rest of the object's properties should be equal)